app/src/app/state.js

```diff
@@ -8,5 +8,5 @@ state.getUrl = function getUrl() {
   return `${this.host.protocol}://${this.host.domain}/${this.host.path}`
 }
 
-state.getUrl = state.getUrl.bind(state)
+state.getUrl = state.getUrl.bind(clone(state))
```

Comment:
for some reason state is mutated at some point and the function above doesn't have access to the original state.